Disaster Recovery/SafeHaven 4/Register-DataCenters-SRN-and-Peer-SRN.md

```diff
@@ -0,0 +1,128 @@
+{{{
+  "title": "Register Data Center,SRN and pair SRN's",
+  "date": "05-08-2017",
+  "author": "Mahima Kumar",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Article Overview
+
+This article focusses on how to:
+1) Register Data Centers and SRN's within the SafeHaven Console for CenturyLink Cloud, VMware vSphere and Manual(Standalone VMware ESXi hosts and DCC) datacenter type.

```

Comment:
Change numbering from `1)` to `1.`
This will make this KB consistent with the rest of the KBs.